### PR TITLE
Minor telegram tweak (km_walked truncated to 2 decimals)

### PR DESCRIPTION
**Pull request by Gobberwart**
## Short Description:

Minor tweaks to telegram

- Truncate km_walked to 2 decimals
- Issue with recognising "master"

### CODE REVIEW

File: pokemongo_bot/event_handlers/telegram_handler.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def run(self):
                     self.update_id = update.update_id+1
                     if update.message:
                         self.bot.logger.info("message from {} ({}): {}".format(update.message.from_user.username, update.message.from_user.id, update.message.text))
-                        if self.master and self.master not in [update.message.from_user.id, "@{}".format(update.message.from_user.username)]:
```
Comment (solderzzc) [inline pokemongo_bot/event_handlers/telegram_handler.py]: Original code has two way to config the master:
username or chat_id, you changed to only chat_id.

**Comment by DBa2016** [comment]
Agreed with @solderzzc here, no need to remove username recognition. The piece which automatically recognized chat_id was removed in the PR which moved Telegram into thread, but will be re-added once #4969 is merged.

**Comment by Gobberwart** [comment]
I removed it because the original didn't work.

File: pokemongo_bot/event_handlers/telegram_handler.py (hunk 1 above)
Comment (Gobberwart) [inline pokemongo_bot/event_handlers/telegram_handler.py]: I see what you're saying, but it wasn't responding to my /info at all with the way it was.. Will keep looking.

**Comment by Gobberwart** [comment]
Looks like I had a telegram api issue while testing this. Reverted to original because it's working fine.

The only change here is truncating km to 2 decimals. Should be a no-brainer.

**Comment by solderzzc** [comment]
yep, easy to merge this one:)

**Comment by solderzzc** [comment]
👍

File: pokemongo_bot/event_handlers/telegram_handler.py (hunk 1 above)
Comment (askovpen) [inline pokemongo_bot/event_handlers/telegram_handler.py]: Username not work. Master can be userid or @channelname.

Comment (Gobberwart) [inline pokemongo_bot/event_handlers/telegram_handler.py]: I just put this back the way it was. If the original isn't working, fix it and create a new PR

**Comment by DBa2016** [comment]
The recognition of non-numeric master is not working right now because #4951 has removed the relevant piece of code without replacement.

**Pull request merged**